testing_and_setup/compass/ocean/global_ocean/SO60to10ISC/init/define_base_mesh.py

```diff
@@ -0,0 +1,46 @@
+# /usr/bin/env python
+"""
+% Create cell width for this mesh on a regular latitude-longitude grid.
+% Outputs:
+%    cellWidth - m x n array, entries are desired cell width in km
+%    lon - longitude, vector of length m, with entries between -180 and 180, degrees
+%    lat - latitude, vector of length n, with entries between -90 and 90, degrees
+"""
+import numpy as np
+import jigsaw_to_MPAS.mesh_definition_tools as mdt
+
+
+def cellWidthVsLatLon():
+    lat = np.arange(-90, 90.01, 0.1)
+    lon = np.arange(-180, 180.01, 10.0)
+
+    cellWidth10 = 10.0 * np.ones(lat.size)
+    cellWidth30 = 30.0 * np.ones(lat.size)
+
+    latTransition = -40.0
+    latWidthTransition = 5.0
```

Comment:
I decided to only add a single case for SO60to10ISC. The specification of the location and width of the southern transition is here, so it is easy enough to alter.

Reply:
![image](https://user-images.githubusercontent.com/4036816/61309398-61985780-a7af-11e9-9a7d-32175e4669e3.png)